README.md

```diff
@@ -1,93 +1,284 @@
 
-The initial Project Team for projects in the `conda` organization are specified in the vote to move a project from the incubator to the main `conda` organization. As this vote has a 60% threshold and sub-team formation is only a greater than 50% threshold, the vote for adding a project to the main `conda` organization can serve both purposes. Project Teams for incubating projects do not need a vote for their creation. Instead they are created when the project is moved into `conda-incubator`.
+The initial Project Team for projects in the `conda` GitHub organization are specified in the vote to move a project from the incubator to the main `conda` GitHub organization. As this vote has a 60% threshold and sub-team formation is only a greater than 50% threshold, the vote for adding a project to the main `conda` GitHub organization can serve both purposes. Project Teams for incubating projects do not need a vote for their creation. Instead they are created when the project is moved into `conda-incubator`.
+
+For [federated](#federated-projects) and [incubating](#incubation) projects, the Project Team decides among themselves on how to add and/or remove members from the Project Team. Further, Project Teams for incubating projects are exempted from the requirement to have at least one Steering Council member. For federated projects, the Project Team must have at least one Steering Council member. For a community maintained project, a petition for commit rights is made to the current members of the Project Team for that project. If this petition passes according to the voting rules for "[Nominate new member of a Project Team](#nominate-new-member-of-a-community-project-team) below, the petitioner is granted write access and added to the team.
+
+## Steering Council Membership
+
+Steering Council members need to 1) be invested in the conda ecosystem and community, and 2) have time and energy to contribute to conda Organization governance.  Council members, with the exception of [provisional members](#provisional-members), will have a demonstrated record of prior service to the conda community.
+
+### Steering Council Size
+
+The Steering Council should reflect a wide range of interests, and still be small enough to have a reasonable expectation that [quorum requirements](#quorum) can be met when votes are called.
+
+The minimum and maximum sizes for the Steering Council are:
+
+* Minimum: 9
+* Maximum: 21
+
+The minimum of 9 works well with the [minimum quorum size of 5](#quorum) specified below. The maximum of 21 allows for a diversity of members, including as many as 7 Provisional Members (see below), and representation from 11 to 21 different participating organizations (see below).
+
+If the Steering Council falls below its minimum, then all other Council business is suspended until this requirements is met. The requirement can be met by adding Provisional members, adding new contributing members, or members coming back from Emeritus status.
```

Comment:
> I think `all other Council business` is a bit vague. This most likely needs to call out explicitly what business can be performed, eg the election of new members.

Hmmm.  Every way I came up with to rewrite this statement:

`If the Steering Council falls below its minimum, then all other Council business is suspended until this requirements is met.`

actually made things *less* clear.  The best I came up with was:

`If the Steering Council falls below its minimum, then the Steering Council's first priority is to meet this requirement.`

Which actually says something slightly different.  :-(

@CJ-Wright (or anyone), any suggestions would be most welcome.

Reply:
> I'll also note that the quorum size would imply that elections of members to the Council would not be possible and so would effectively grind operations to a halt permanently.

I had not thought about the possibility of dropping below 5 Steering Council members.  That would require first dropping to 9 members, and then the mass simultaneous resignation of at least 5 members (a majority of the remaining council).

Unfortunately there are good reasons for that minimum quorum size of 5.  If we modify that requirement it would take a paragraph of text and more rules to explain an equivalent policy for minimum quorum size.

**@CJ-Wright, can I talk you into dealing with this situation if and when we get close to it?**  If we ever get close to just 9 members, then that implies that we need to reconsider a lot more than just minimum quorum size.  If we get close to 9 members then this whole approach needs to be revamped.

Reply:
We should not leave things to be resolved. They should be explicit and decided now. We have to let the minimum quorum rule be overrides by the vote timeout rules. If after a total of 3 emails, 3 chat messages, and a discussion at a community meeting over two weeks not more than five people respond to a vote then I think it should be fine for people to go ahead.

Reply:
In the topic of collaboration council business allowed if we do not meet membership requirements, clearly the only actions allowed should be votes related to changing the council membership. These are enumerated in the new doc already so we should simply list them.

Reply:
@CJ-Wright We dropped the maximum number of council members from the proposal.  This will help address the concern about when someone leaves a company, but not the council.